src/configurations/destinations/intercom_v2/db-config.json

```diff
@@ -20,10 +24,11 @@
       "cloud",
       "reactnative",
       "flutter",
-      "cordova"
```

Comment:
Do we want to support all these sourceTypes for intercom v2 and not just warehouse?

Reply:
we are supporting eventstream and retl both.